_posts/2020-02-10-q1-2020-update.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Bisq Q1 2020 Update: Reorganization"
+author: Chris Beams
```

Comment:
Let's make this `Chris Beams and Steve Jain` please. I changed my mind about literally ghost writing this. Much of the underlying content is mine, but you did the work of writing it, and there are stylistic differences to what I would have written if I'd done it myself (all of which are fine, I just want to avoid the dissonance that attributing it fully to me might cause). And it's just more transparent that way. I double checked the jekyll arrangement, and there should be no problem literally just adding ` and Steve Jain` to the `author:` metadata.